.gitignore

```diff
@@ -31,6 +31,7 @@ node_modules
 core_keys
 users
 webhooks
+test/__test_data__/*
```

Comment:
if i do ` path.join(__dirname, '../test/__test_data__/users'),` and  `__test_data__` doesn't exist and `users` dir doesn't exist.. node throws an error...
so I create `__test_data__` dir by hands and ignore all inside it.
I could't find a way to make node clever enough to create deep not existing dirs automatically with path.join() :(